api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -51,14 +50,13 @@ message UpstreamTlsContext {
   // interacts with other validation options.
   bool auto_host_sni = 6;
 
-  // If true, replace any Subject Alternative Name validations with a validation for a DNS SAN matching
-  // the SNI value sent. Note that the validation will be against the actual requested SNI, regardless of how it
-  // is configured.
+  // If true, replaces any Subject Alternative Name (SAN) validations with a validation for a DNS SAN matching
+  // the SNI value sent. The validation uses the actual requested SNI, regardless of its configuration.
```

Comment:
I think the final sentence rewording here makes it less clear. There are 3-4 different ways the final SNI can be computed via different configuration and runtime, so `its` here doesn't have a clear antecedent.

Reply:
Will change it to say, _"The validation uses the actual requested SNI, regardless of how the SNI is configured."_